sql_generators/glean_usage/templates/baseline_clients_first_seen_v1.init.sql

```diff
@@ -34,12 +34,13 @@ WITH
   {{ core_clients_first_seen(migration_table) }}
   SELECT
     client_id,
-    submission_date,
-    COALESCE(core.first_seen_date, baseline.first_seen_date) as first_seen_date,
-    sample_id
+    MIN(submission_date) AS submission_date,
+    MIN(COALESCE(core.first_seen_date, baseline.first_seen_date)) AS first_seen_date,
+    sample_id,
   FROM baseline
   LEFT JOIN _core_clients_first_seen core
   USING (client_id)
+  GROUP BY client_id, sample_id
```

Comment:
isntead of adding this here, we should ensure that we aren't getting duplicates from `_core_clients_first_seen`

Reply:
what we could do here is have the query error out if there's a duplicate, because the inputs to this join must be unique on client id,

Reply:
Do you mean we should ensure a single row is returned here:
https://github.com/mozilla/bigquery-etl/blob/0528cbe6c5ba3205e14e9fb2115a9641a260828b/sql_generators/glean_usage/templates/macros.sql#L21-L30
?

Reply:
@relud could you please share how you'd approach this? I'd like to close down related issue.

Reply:
yeah, that seems like a reasonable place to put the check